src/webprojects/fxa.md

```diff
@@ -0,0 +1,116 @@
+* _Subscription management UI_: Settings that allow the user to update their payment information.
+
+Payment testing is tied to the language of user preference and location. Payment service is market specific. For instance, the product is available in German, but the service is available in Germany but not in Switzerland.
+
+#### Before testing
+
+* Have a Firefox account. If you don’t have one, the staging link will take you to the signup process.
+* Choose a Firefox browser for: desktop (Windows only), Android, or iOS. Support for macOS and Linux is coming soon.
+* If you are not already using the browser in the language you are going to test in, type in the URL field `about:Preferences`. Or follow the steps detailed in this [SUMO article](https://support.mozilla.org/en-US/kb/choose-display-languages-multilingual-web-pages) to configure to the correct display language.
+* Have the staging link ready for [subcription testing](accounts.stage.mozaws.net/subscriptions/products/prod_HEJ13uxjG4Rj6L?plan=plan_HEJ1l12p1taV7I)
+
+**Tips:** Feeling not having enough test accounts to use for testing? You are not alone. Here are some tips you can consider. Note that this works with Gmail and Google Apps domains:
+
+* Add number(s) or letter(s) to your existing account to create more accounts by adding a `+` in your email address. For example, if your main email is `name@xyz.com`, your expanded accounts could be `name+1@xyz.com` and `name+m@xyz.com`. You will receive email notifications in your `name@xyz.com` inbox. You can add anything you want, several digits and/or letters.
+* Keep your password simple so you can log in and log out easily. Keep track of which account is connected with which password. Of course, there is Lockwise for the rescue!
+
+#### Testing payment and subscription creation
+
+* Go to the [subscription staging server](accounts.stage.mozaws.net/subscriptions/products/prod_HEJ13uxjG4Rj6L?plan=plan_HEJ1l12p1taV7I)
+* Click on the **Subscribe** button which takes you to a subscription form. If you don't have a Firefox account at this point, you will need to create one.
+* Fill out all the required fields with any made up information you want, as long as the format conforms to the requirement for your market.
+* Type credit card number `4242 4242 4242 4242`.
+* Check formatting accuracy for date, currency and postal code. Check the box on usage agreement.
+* Click on **Pay Now**
+* An email confirmation arrives in your inbox. The content should match the summary on the page where you just completed your subscription.
+* A *Thank You* payment confirmation message appears with the subscription summary on your browser page with an invitation to download. Click on **Continue to Download**.
+* It takes you to the Firefox Privacy Network (FPN) add-on download page. Click on **+ Add to Firefox** button. A series of pop-up windows will walk you through the installation process, and where to find and manage your FPN.
```

Comment:
I think all the steps about installing the add-on should be dropped.

* The FPN add-on is not localized.
* I created an account on the stage server, that's not available here, because the add-on uses the production server to log me in.
* I logged in with my normal FxA account, and it doesn't work.
* I expect the add-on to be geo-blocked.

If the goal is to manage your account settings, you should stick to the staging server, without adding the add-on.

And you need to use the server you used for the test plan (https://accounts.stage.mozaws.net), not the staging server you mention at the beginning, because that won't work…

Do we have any guarantee that this server will stay?

Reply:
Good point on the FPN. 

The subscription test link will stay but I am not sure for how long. I am waiting for a definite answer on this. I suspect this is a temporary solution to get this going. The regular FxA staging link remains